doc/nrf/drivers/wifi/nrf700x/nrf700x.rst

```diff
@@ -156,12 +156,4 @@ To use the Ethernet interface, the application can use `Zephyr Network APIs`_.
 Low-level API documentation
 ***************************
 
-The nRF70 Series Wi-Fi driver provides a low-level API for use cases where the application needs to access the nRF70 Series device directly.
-This is typically intended for customers who want to use the nRF70 Series device in a different platform than Zephyr.
-
-.. toctree::
-   :maxdepth: 1
-   :caption: Modules:
-   :glob:
-
-   subpages/*
+Refer to the :ref:`nrfxlib:nrf_wifi_api` for the low-level API documentation.
```

Comment:
Since the modules have moved from here, can we add this line under "API documentation" section and remove "Low-level API documentation" section? 
```suggestion
See the :ref:`nrfxlib:nrf_wifi_api` to learn more about various modes of low-level API.
```